### PR TITLE
use merge_files and hipcc in packaging

### PR DESCRIPTION
**Pull request by ce1adon**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -50,22 +50,31 @@ if(BUILD_DEV)
     set(MIOPEN_TENSILE_SRC dev)
 endif()
 # Use the virtual-env setup and download package from specified repo:
-set( MIOPEN_TENSILE_TAG 698e3174c4d6e5045e9a619158cd075273ccef96 CACHE STRING "Tensile tag to download" )
+set( MIOPEN_TENSILE_TAG 3c25a6099b94549c5d6a8c4edb7deba617e22c71 CACHE STRING "Tensile tag to download" )
 virtualenv_install(wheel)
-virtualenv_install("git+https://github.com/ROCmSoftwarePlatform/Tensile.git@${MIOPEN_TENSILE_TAG}")
+virtualenv_install("git+https://github.com/ce1adon/Tensile.git@${MIOPEN_TENSILE_TAG}")
```
Comment (pfultz2) [inline CMakeLists.txt:55]: Is there a reason to pull from you fork? The change should be pushed to ROCmSoftwarePlatform.

Comment (ce1adon) [reply CMakeLists.txt:55]: @pfultz2 Tensile has issue with this commit and they suggested to revert it https://github.com/ce1adon/Tensile/commit/707d7dcbd53515f17cefd8bc00728a86c1c45152

Comment (ce1adon) [reply CMakeLists.txt:55]: Once this PR is checked in, we can change the dependency to rocmplatform.
https://github.com/ROCmSoftwarePlatform/Tensile/pull/1057

Comment (ce1adon) [reply CMakeLists.txt]: Updated version with latest tensile develop commit.

**Comment by ce1adon** [comment]
@pfultz2 I have tested this PR on Centos 7/8 and it works. Can you also review it?

**Pull request merged**